test/lint/lint.js

```diff
@@ -22,14 +19,34 @@ const cli = new CLIEngine({
 
 cli.addPlugin('eslint-plugin-unicorn', unicorn);
 
+// Find a way to make sure rules are loaded from codebase
+
 const report = cli.executeOnFiles(files);
 
-if (fix) {
+const {errorCount, warningCount, fixableErrorCount, fixableWarningCount} = report;
+
+const hasFixable = fixableErrorCount || fixableWarningCount;
+
+if (fix && hasFixable) {
 	CLIEngine.outputFixes(report);
 }
 
-const formatter = cli.getFormatter();
+if (errorCount || warningCount) {
+	const formatter = cli.getFormatter();
+	console.log(formatter(report.results));
 
-console.log(formatter(report.results));
+	console.log();
+	console.log('Some tests have failed, you need fix them and run `npm run lint <file>` to check again.');
```

Comment:
what about `Not all tests have passed`?
Just a thought since it can be only one test that failed

Reply:
haha, okay. I can also know how many (errorCount + warningCount)

Reply:
@yakov116

https://travis-ci.org/sindresorhus/eslint-plugin-unicorn/jobs/604272880#L225

good?

Reply:
Perfect!